src/test/java/duke/task/TodoTest.java

```diff
@@ -0,0 +1,23 @@
+package duke.task;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class TodoTest {
+
+    @Test
+    public void toStringTest() {
```

Comment:
Perhaps this should be testToString()?

Reply:
I agree, that is a better naming convention